low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -463,9 +465,61 @@ def process(self):
 
 
 class CleanupCmd(Cmd):
-    """Removes SSPL configs."""
+
+    """Restores the default SSPL configs."""
 
     name = "cleanup"
+    product = None
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def validate(self):
+        # Validate config inputs
+        from framework.utils.utility import Utility
+        Conf.load(GLOBAL_CONFIG_INDEX, global_config_path)
+        self.product = Utility.get_config_value(GLOBAL_CONFIG_INDEX,
+            "cortx>release>product")
+        if self.product is None:
+            msg = "%s - validation failure. %s" % (
+                self.name, "'Product' name is required.")
```

Comment:
'Product' name is required to restore suitable configs.

Reply:
Done